### ISSUE TITLE
Failed resolution of: Landroidx/compose/ui/platform/AndroidAmbientsKt;

### ISSUE THREAD

**Issue by piotrsedlak** [skydoves/Landscapist#13]
**Please complete the following information:**
- Library Version 1.1.5 
 
**Describe the Bug:**

After updating to compose beta01, when loading image i receive a crash(tried on coil and glide):

```
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            TestmobileTheme {
                // A surface container using the 'background' color from the theme
                Surface(color = MaterialTheme.colors.background) {
                    Box(){
                    CoilImage( imageModel = "https://user-images.githubusercontent.com/24237865/94174882-d6e1db00-fed0-11ea-86ec-671b5039b1b9.gif", modifier = Modifier.fillMaxSize())
                    }
                }
            }
        }
    }
}
```

```
 java.lang.NoClassDefFoundError: Failed resolution of: Landroidx/compose/ui/platform/AndroidAmbientsKt;
        at com.skydoves.landscapist.coil.CoilImage__CoilImageKt.CoilImage(CoilImage.kt:161)
        at com.skydoves.landscapist.coil.CoilImage.CoilImage(CoilImage.kt:1)
        at com.example.testmobile.ComposableSingletons$MainActivityKt$lambda-1$1.invoke(MainActivity.kt:41)
        at com.example.testmobile.ComposableSingletons$MainActivityKt$lambda-1$1.invoke(MainActivity.kt:39)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.kt:121)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.kt:46)
        at androidx.compose.material.SurfaceKt$Surface$1.invoke(Surface.kt:116)
        at androidx.compose.material.SurfaceKt$Surface$1.invoke(Surface.kt:105)


    Caused by: java.lang.ClassNotFoundException: Didn't find class "androidx.compose.ui.platform.AndroidAmbientsKt" on path: DexPathList[[zip file "/data/app/~~7Vhvt_5Kd29kOwyJzsopIA==/com.example.testmobile-Fu0vMSGdvFCsARea7UE3jA==/base.apk"],nativeLibraryDirectories=[/data/app/~~7Vhvt_5Kd29kOwyJzsopIA==/com.example.testmobile-Fu0vMSGdvFCsARea7UE3jA==/lib/x86, /system/lib, /system_ext/lib]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:207)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        	... 88 more
```

[crash.txt](https://github.com/skydoves/Landscapist/files/6043778/crash.txt)

**Expected Behavior:**
Image should load

**Comment by skydoves** [comment]
Beta01 will be supported in the next release.
Thanks!

**Comment by skydoves** [comment]
We can available now the new release `1.1.6` with the compose `1.0.0-beta01`!

**Comment by Sirelon** [comment]
Thank you for your quick update! Works good!
